tests/apitests/python/test_scan_data_export.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+import time
+
+import unittest
+
+from testutils import harbor_server, suppress_urllib3_warning
+from testutils import ADMIN_CLIENT
+from library.scan_data_export import Scan_data_export
+from library.project import Project
+from library.user import User
+from library.artifact import Artifact
+from library.scan import Scan
+from library.repository import push_self_build_image_to_project
+
+class TestScanDataExport(unittest.TestCase):
+
+    @suppress_urllib3_warning
+    def setUp(self):
+        self.scan_data_export = Scan_data_export()
+        self.project = Project()
+        self.user = User()
+        self.scan = Scan()
+        self.artifact = Artifact()
+        self.image = "alpine"
+        self.tag = "latest"
+        self.x_scan_data_type = "application/vnd.security.vulnerability.report; version=1.1"
+
+    def testScanDataExportArtifact(self):
+        """
+        Test case:
+            Scan Data Export API
+        Test step and expected result:
+            1. Create a new user(UA);
+            2. Create a new project(PA) by user(UA);
+            3. Push a new image(IA) in project(PA) by user(UA);
+            4. Send scan image command and get tag(TA) information to check scan result, it should be finished;
+            5. Verify trigger export scan data execution but does not specify Scan-Data-Type status code should be 422;
+            6. Verify trigger export scan data execution but specifying multiple project status code should be 400;
+            7. Trigger export scan data execution correctly;
+            8. Verify that the export scan data execution triggered by the user(UA) cannot be queried by other users;
+            9. User (UA) should be able to query the triggered export scan data execution;
+            10. Wait for the export scan data execution to succeed;
+            11. Verify that the export scan data execution triggered by the user (UA) cannot be download by other users;
+            12. User (UA) should be able to download the triggered export scan data execution
```

Comment:
Do we need to check the correctness of the content in the CSV file?

Reply:
The correctness of the CSV file will be checked in the RobotFramework test case, because the push image in the python test case actually has no CVE, so the correctness of the CSV file cannot be checked.

Reply:
ok, got it.